commit 6ba83321f5e573b80e6941587560ec0162b0323a (refs/changes/11/3311/1)
Author: David Pacheco <dap@joyent.com>
Date:   2018-02-07T17:05:40-08:00 (1 year, 8 months ago)
    
    MORAY-455 moray asks fast to hold onto too many recent RPC requests

diff --git a/lib/client.js b/lib/client.js
index 50bab99..1a2a6da 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -35,7 +35,7 @@ var parseMorayParameters = require('./client_params').parseMorayParameters;
 
 ///--- Default values for function arguments
 
-var fastNRecentRequests         = 30;
+var fastNRecentRequests         = 5;
 var dflClientTcpKeepAliveIdle   = 10000;  /* milliseconds */
 
 /*
